src/Tizen.Network.Nsd/Tizen.Network.Nsd/DnssdService.cs

```diff
@@ -218,14 +220,33 @@ public IPAddressInformation IP
             }
         }
 
-        private void GetTxtRecord(out ushort length, out byte[] value)
+        /// <summary>
+        /// Returns all raw TXT records.
+        /// </summary>
+        /// <remarks>
+        /// After adding the TXT record call for get TXT record.
```

Comment:
I'm not sure I get it. If you meant that when the user sets TXT, she then can get the value that was set, then I we could even drop that sentence as quite obvious. Optionally add `<seealso cref="SetRecord(ushort, byte[])" />` tag.

Reply:
Fixed